packages/map-template/src/components/WebComponentWrappers/Search/Search.jsx

```diff
@@ -112,6 +118,7 @@ const SearchField = forwardRef(({ placeholder, mapsindoors, results, clicked, cl
         disabled={disabled}
         mapbox={mapbox}
         google={google}
+        mi-venue={kioskLocation && isKioskContext ? kioskLocation.properties.venueId : undefined}
```

Comment:
Add a comment explaining that when the Map Template runs in kiosk mode, we restrict searching to the venue only.